extension/lib/tools.js

```diff
@@ -0,0 +1,47 @@
+class Tools {
+  /**
+   * @property {Object} path.storage.device
```

Comment:
Why is this here? `@property {Object} path.storage.device`

Reply:
Just copied the file from the shopify-cart extension, deleted this